pint/models/timing_model.py

```diff
@@ -96,6 +97,21 @@ def use_cached_results(*args, **kwargs):
         return use_cached_results
 
 
+class ModelMeta(abc.ABCMeta):
+    """
+    This is a Meta class for timing model registeration. In order ot get a
+    timing model registered, a member called 'name' has to be provided in a
```

Comment:
The docstring here still says "name", but it's now "is_register" I think.

"is_register=True" sounds a little off to me, because it's a mix between "is_registered" and "register me". Maybe call it "register"?